app/models/manageiq/providers/ansible_tower/shared/inventory/parser/configuration_script_source.rb

```diff
@@ -0,0 +1,8 @@
+module ManageIQ::Providers::AnsibleTower::Shared::Inventory::Parser::ConfigurationScriptSource
+  include parent::AutomationManager
```

Comment:
this is really nice, I just need to override the `collector` and the `parse` method in here